paths/custom_field.yaml

```diff
@@ -0,0 +1,53 @@
+get:
+  operationId: custom-field
+  tags: [ custom_fields ]
```

Comment:
custom-fields / custom-field?

Reply:
custom_fields is a tag related to all custom fields api

Reply:
I suggest avoiding `_` and use `-` as a separator.